src/simtools/applications/run_application.py

```diff
@@ -111,13 +111,10 @@ def read_application_configuration(configuration_file, logger):
     application_config = gen.collect_data_from_file(configuration_file).get("CTA_SIMPIPE")
     place_holder = "__SETTING_WORKFLOW__"
     workflow_dir, setting_workflow = get_subdirectory_name(configuration_file)
-    output_path = str(workflow_dir).replace("input", "output") + setting_workflow
+    output_path = str(workflow_dir).replace("input", "output") + "/" + setting_workflow
```

Comment:
I assume workflow_dir is already a `Path`. In which case, it would be nicer to do

`output_path = workflow_dir.with_name(workflow_dir.name.replace("input", "output")) / setting_workflow`

This gets rid of string manipulations of paths (and would return a `Path` which would mean that in the following line you can remove the `Path` definition, but it doesn't hurt).

Reply:
Not about number of characters, but about not performing string operations on paths.